bin/perseo

```diff
@@ -157,7 +158,7 @@ function loadConfiguration() {
     config.smpp.enabled = process.env.PERSEO_SMPP_ENABLED || config.smpp.enabled;
     config.endpoint.noticesPath = process.env.PERSEO_NOTICES_PATH || config.endpoint.noticesPath;
     config.endpoint.rulesPath = process.env.PERSEO_RULES_PATH || config.endpoint.rulesPath;
-
+    config.castTypes = process.env.PERSEO_CAST_TYPES || config.castTypes;
```

Comment:
Should the new config parameter / env var to be documented in some .md file? Not remember how we are doing this in Perseo...

Reply:
Done in 66142b24de3cd86bdb113738e69543a32665c7ab